BrillouinAcquisition/src/Devices/ScanControls/ZeissMTB.cpp

```diff
@@ -79,11 +79,11 @@ void ZeissMTB::setPosition(POINT2 position) {
 	auto positionStage = position - m_positionScanner;
 	if (abs(m_positionStage.x - positionStage.x) > 1e-6) {
 		m_positionStage.x = positionStage.x;
-		success = m_stageX->SetPosition(m_positionStage.x, "�m", MTBCmdSetModes::MTBCmdSetModes_Synchronous, 500);
+		success = m_stageX->SetPosition(m_positionStage.x, "�m", MTBCmdSetModes::MTBCmdSetModes_Synchronous, 500);
```

Comment:
These changes will very likely break setting positions for Zeiss stages.

Reply:
Thank you @raimund-schluessler 

I do not see the bigger picture here. Could you please elaborate? From my perspective, this looks like an encoding issue. I am also not sure whether we are able to test this @RaghavaAlajangi ?